src/cavendish_particle_tracks/_stereoshift_dialog.py

```diff
@@ -156,11 +156,35 @@ def _setup_stereoshift_layer(self):
             ]
         )
 
+        points = np.array(
+            [
+                [
+                    origin_x - 100 / zoom_factor,
+                    origin_y - 200 / zoom_factor,
+                ],
+                [
+                    origin_x + 100 / zoom_factor,
+                    origin_y - 200 / zoom_factor,
+                ],
+                [origin_x - 100 / zoom_factor, origin_y],
+                [origin_x + 100 / zoom_factor, origin_y],
+                [
+                    origin_x - 100 / zoom_factor,
+                    origin_y + 200 / zoom_factor,
+                ],
+                [
+                    origin_x + 100 / zoom_factor,
+                    origin_y + 200 / zoom_factor,
+                ],
+            ]
+        )
+
```

Comment:
Should this be in here? It's just appeared in this PR.

Reply:
Oops 😊